libs/blocks/quiz/quiz.css

```diff
@@ -62,44 +62,48 @@
 }
 
 .quiz-option {
-  align-items: center;
+  align-items: stretch;
```

Comment:
Does this cause layout shift?

Reply:
@fullcolorcoder So I've been testing a bunch of changes manually using a test branch that was created from this PR's branch. I notice no significant difference at all when I've set this back to center. I think this isn't having any impact. I've tried a couple of other things, like removing the background-color code altogether - no difference at all. I have noticed that when I get lower scores, I'm getting TBT from typekit, though.